assets/js/phoenix_live_view.js

```diff
@@ -1535,7 +1535,7 @@ class DOMPatch {
   }
 
   forceFocusedSelectUpdate(fromEl, toEl){
-    return fromEl.multiple === true || fromEl.innerHTML != toEl.innerHTML
+    return fromEl.multiple === true || (fromEl.type === "select" && fromEl.innerHTML != toEl.innerHTML)
```

Comment:
I am not 100% sure about this but it seems that the issue has been fixed by this. And, as far as I get from this function, this is specific for `selects` input, that is why I've added this check 🙌 .

Basically, when the input is textarea, the `innerHTML` will always be different. This is not true for normal text inputs because `innerHTML` always returns `""`.

Reply:
@chrismccord  @feliperenan This patch seems to break updating select elements when they are focuses see: #1313. Any suggestions on how to tackle this so I can provide some assistance.